Source/Layout/ASYogaUtilities.mm

```diff
@@ -213,8 +213,8 @@ YGSize ASLayoutElementYogaMeasureFunc(YGNodeRef yogaNode, float width, YGMeasure
   id <ASLayoutElement> layoutElement = (__bridge id <ASLayoutElement>)YGNodeGetContext(yogaNode);
   ASDisplayNodeCAssert([layoutElement conformsToProtocol:@protocol(ASLayoutElement)], @"Yoga context must be <ASLayoutElement>");
 
-  width = cgFloatForYogaFloat(width);
-  height = cgFloatForYogaFloat(height);
+  width = cgFloatForYogaFloat(width, CGFLOAT_MAX);
+  height = cgFloatForYogaFloat(height, CGFLOAT_MAX);
```

Comment:
Is there a case where the undefined default is something besides CGFLOAT_MAX?

Reply:
good point. I think it's cleaner to stick to `CGFLOAT_MAX` as the default value (and thus no need for the extra `undefinedDefault` param) until we have cases in which `CGFLOAT_MAX` is not desired.

Reply:
There will be a case where `CGFLOAT_MAX` is not desired ...